http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
++
+** *Query Parameters*
++
+Query parameters to include in the request.
++
+* *Correlations ID*
++
+Select an option for whether to include an outbound correlation ID in the request. 
++
+. Select the *Advanced* tab to configure the following attributes:
++
+* *Output MIME type*
++
+Defines the format of the request.
++
+* *Parameters*
++
+Add parameters to the request. 
++
+* *Output encoding*
++
+Select the encoding to transform the data to.
++
+* *Streaming Strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:connectors::streaming-strategies.adoc[]
++
+* *Connector Overrides*
+** *Follow redirects*
++
+Specifies whether or not to follow redirects.
++
+** *Send body mode*
++
+Select an option to define whether the request contains a body. If you select *Auto*, the behavior is determined by the defined allowed HTTP methods (GET, HEAD, and OPTIONS does not send a body).
++
+** *Request streaming mode*
++
+Select an option to define whether the request is sent using streaming.
++ 
+** *Response timeout*
++
+Maximum time in milliseconds that the request element blocks the execution of the flow waiting for the HTTP response.
++
+* *Response*
+** *Response validator*
++
+Configures default error handling for the request. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#SuccessStatusCodeValidator[Success Status Code Validator] and xref:http-documentation.adoc#FailureStatusCodeValidator[Failure Status Code Validator].
+* *Output*
+** *Target Variable*
++
+Name of the variable on which the operation's output is placed.
++
+** *Target Value*
++
+An expression that's evaluated against the operation's output. The outcome of that expression is stored in the target variable.
++ 
+* *Error Mappings*
++
+This attribute is not currently enabled. 
++
+* *Connection*
+** *Reconnection strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#reconnect[Reconnect] and xref:http-documentation.adoc#reconnect[Reconnect Forever]. 
+
+== Configuring Attributes for Load Static Resource
+
+The static resource serves static content to use with HTTP. The request path is used to look up the resource. 
+
+To configure the attributes for HTTP *Load static resource* in the Anypoint Code Builder UI:
+
+. Click the *Load static resource* component on the canvas.
++
+The configuration UI opens.
++
+. In the *General* tab, configure the following attributes:
+* *Resource base path*
++
+The base path of the resource that serves up the content, for example, `/Users/maxthemule/` resources.
+* *Default file*
++
+The default file to serve when a directory is specified. The default is `index.html`.
++
+* *Attributes*
++
+The HTTP request attributes coming from the HTTP Listener source to check the required resources. 
```

Comment:
Again, the attributes check the resources? 
Or is it "to specify how to check"?

Reply:
So you define the attributes in the HTTP Listener and you define an attribute to check the Authorization header. I reworded. Hopefully it's more clear now.